<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>heat_equation</class>
 <widget class="QWidget" name="heat_equation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>heat_equation</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>dimension: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>numerical scheme</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>time step: k</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="time_step">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>spatial step: h </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="space_step">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>280</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>total time: T</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="total_time">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>280</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="goButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>370</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>SUBMIT</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="output_file_name">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>320</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>320</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>output file name:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_dim">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>dimension:</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_num_scheme">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>numerical method:</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>INPUT PARAMETERS</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>goButton</sender>
   <signal>clicked()</signal>
   <receiver>heat_equation</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>374</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
